README.md

```diff
@@ -201,13 +228,13 @@ Baumeister acts like a static sites generator by default. Using handlebars we ca
 ### This is optional
 Using Handlebars instead of plain HTML is fully optional and will probably suit your needs if you use Baumeister for creating a static site. If you are developing a single page application instead you might turn off handlebars compiling and place just an `index.html` file in the `/src` directory and store additional templates in `/src/app`.
 
-In this case you have to switch off Handlebars compiling in `gulp/config.js`:
+In this case you have to switch off Handlebars compiling in `baumeister.json`:
 
 ```javascript
```

Comment:
May we use JSON syntax here?

Reply:
Used JavaScript Syntax to properly render the comments ;)